SU-HSDO/suhumsci
# fix: fix missing default image for site manager role

# READY FOR REVIEW

## Summary
This PR adds a CSS fix to allow for contextual default image placeholders to be displayed for the Site Manager role. Currently, the contextual image placeholder `<div>` is being pushed below another `<div>` for this role, as it is missing a needed `contextual` class. This PR adds 'position: absolute` to the placeholder to simulate the existence of the `contextual` class.

JIRA Card: https://sparkbox.atlassian.net/browse/STN-975

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. Use the Masquerade module to simulate logging in as a user with the Site Manager role (ex: Stefanie Pietkiewicz)
2. Navigate to the News page and locate an article with a default image.
3. Verify that the default image is visible for this user
4. Verify that other news article images are unaffected by this update, switching to other user roles as well.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
